includes/class-theme-base.php

```diff
@@ -598,6 +634,15 @@ function post_count() {
 	/**
 	 * Returns a list of objects based on the queried object from $wp_the_query
 	 *
+	 * @return WPLib_Post_List_Default|WPLib_Post_Base[]
+	 */
+	function post_list() {
+		return self::get_post_list();
```

Comment:
`get_post_list()` should not be able to be called without a parameter.  I assume it should be `get_post_list(array())` and then the declaration of `get_post_list()`should not allow the parameter to be optional?